operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md

```diff
@@ -0,0 +1,144 @@
+
+   **WARNING:** If the above script is not run, **DO NOT** download the `initrd` or kernel after the
+   CFS session completes (see below). CFS will collect `/boot/initrd` and `/boot/vmlinuz` from the modified
+   SquashFS. If the `create-ims-initrd.sh` script is not run, then these artifacts will be incorrect and
+   will not be able to boot an NCN.
+
+   When not modifying anything at the kernel-level, there is no need to create a new `initrd`. In
+   that case, simply use the existing `initrd`.
+
+1. Create a CFS configuration.
+
+    See [Create a CFS Configuration](Create_a_CFS_Configuration.md).
+
+1. Create an image customization CFS session.
+
+    See [Create an Image Customization CFS Session](Create_an_Image_Customization_CFS_Session.md).
+
+1. Download the resultant NCN artifacts.
+
+    > **NOTE:** `$IMS_RESULTANT_IMAGE_ID` is the IMS image ID returned in the output of the last command
+    > in the previous step:
+    >
+    > ```console
+    > ncn# cray cfs sessions describe example --format json | jq .status.artifacts
+    > ```
+
+    ```console
+    ncn# cray artifacts get boot-images $IMS_RESULTANT_IMAGE_ID/rootfs kubernetes-<version>-1.squashfs
+
+    ncn# cray artifacts get boot-images $IMS_RESULTANT_IMAGE_ID/initrd initrd.img-<version>-1.xz
+
+    ncn# cray artifacts get boot-images $IMS_RESULTANT_IMAGE_ID/kernel 5.3.18-150300.59.43-default-<version>-1.kernel
+    ```
+
+1. Upload NCN boot artifacts into S3.
+
+    This steps requires that the `docs-csm` RPM is installed. See [Check for Latest Documentation](../../update_product_stream/index.md#documentation).
+
+    ```console
+    ncn# /usr/share/doc/csm/scripts/ceph-upload-file-public-read.py --bucket-name ncn-images --key-name 'k8s/<new-version>/filesystem.squashfs' --file-name kubernetes-<version>-1.squashfs
+
+    ncn# /usr/share/doc/csm/scripts/ceph-upload-file-public-read.py --bucket-name ncn-images --key-name 'k8s/<new-version>/initrd' --file-name initrd.img-<version>-1.xz
+
+    ncn# /usr/share/doc/csm/scripts/ceph-upload-file-public-read.py --bucket-name ncn-images --key-name 'k8s/<new-version>/kernel' --file-name <version>-1.kernel
+    ```
+
+1. Update NCN boot parameters.
+
+    1. Set a variable with the component name (xname) of the node of interest.
+
+        ```console
+        ncn# XNAME=<your-xname>
+        ```
+
+    1. Get the existing `metal.server` setting for the node.
+
+        ```console
+        ncn# METAL_SERVER=$(cray bss bootparameters list --hosts $XNAME --format json | jq '.[] |."params"' \
+             | awk -F 'metal.server=' '{print $2}' \
+             | awk -F ' ' '{print $1}')
+        ```
+
+    1. Verify that the variable was set correctly.
+
+        ```console
+        ncn# echo $METAL_SERVER
+        ````
+
+    1. Update the kernel, `initrd`, and `metal.server` to point to the new artifacts.
+
+        Note: Multiple xnames can be updated at the same time, if desired.
```

Comment:
```suggestion
        **NOTE:** Multiple xnames can be updated at the same time, if desired.
```
This is the formatting in several instances above.

Reply:
It seems this comment was made on out-of-date source.  The verbiage around 'multiple xnames' was changed in several places.